app/src/main/java/edu/uc/haflga/financialpal/AddTransaction.kt

```diff
@@ -4,7 +4,7 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import edu.uc.haflga.financialpal.ui.addtransaction.AddTransactionFragment
 
-class add_transaction : AppCompatActivity() {
+class AddTransaction : AppCompatActivity() {
 
```

Comment:
Changed name to match Kotlin naming conventions:
https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules